src/elements/content-sidebar/activity-feed/annotation-thread/AnnotationThread.js

```diff
@@ -27,6 +28,7 @@ type Props = {
     className: string,
     clientName: string,
     currentUser: User,
+    eventEmitter?: EventEmitter,
```

Comment:
It doesn't look like this prop is used in this component. Can we remove this?

Reply:
I've included it here for future reference as another PR will be created shortly. `AnnotationThread` will reuse the `useAnnotatorEvents` hook and thus it needs to pass eventEmitter object to it.